immuneML/workflows/instructions/SimulationInstruction.py

```diff
@@ -23,7 +23,7 @@ class SimulationInstruction(Instruction):
 
         simulation (Simulation): definition of how to perform the simulation.
 
-        export_formats: in which formats to export the dataset after simulation. Valid formats are class names of any non-abstract class inheriting :py:obj:`~immuneML.IO.dataset_export.DataExporter.DataExporter`. Important note: Pickle files might not be compatible between different immuneML (sub)versions.
+        export_formats: in which formats to export the dataset after simulation. Valid formats are class names of any non-abstract class inheriting :py:obj:`~immuneML.IO.dataset_export.DataExporter.DataExporter`. Important note: Binary files might not be compatible between different immuneML (sub)versions.
```

Comment:
Minor, but I would phrase it like:

Important note: exported datasets in 'ImmuneML' format might not be compatible between different immuneML (sub)versions.

Reply:
updated. it is compatible within version :)